pkg/test/step.go

```diff
@@ -55,57 +55,63 @@ func (t TestStepParameters) GetInt(k string) (int64, error) {
 	return n, nil
 }
 
-// TestStepStatus represent a blob containing status information that the TestStep
+// StepStatus represent a blob containing status information that the Step
 // can persist into storage to support resuming the test.
-type TestStepStatus string
+type StepStatus string
 
-// TestStepFactory is a type representing a function which builds a TestStep.
-// TestStep factories are registered in the plugin registry.
-type TestStepFactory func() TestStep
+// StepFactory is a type representing a function which builds a Step.
+// Step factories are registered in the plugin registry.
+type StepFactory func() Step
 
-// TestStepLoader is a type representing a function which returns all the
-// needed things to be able to load a TestStep.
-type TestStepLoader func() (string, TestStepFactory, []event.Name)
+// StepLoader is a type representing a function which returns all the
+// needed things to be able to load a Step.
+type StepLoader func() (string, StepFactory, []event.Name)
 
-// TestStepDescriptor is the definition of a test step matching a test step
+// StepDescriptor is the definition of a test step matching a test step
 // configuration.
-type TestStepDescriptor struct {
+type StepDescriptor struct {
 	Name       string
 	Label      string
-	Parameters TestStepParameters
+	Parameters StepParameters
 }
 
-// TestStepBundle bundles the selected TestStep together with its parameters as
-// specified in the Test descriptor fetched by the TestFetcher
-type TestStepBundle struct {
-	TestStep      TestStep
-	TestStepLabel string
-	Parameters    TestStepParameters
+// StepsDescriptors bundles together Test and Cleanup descriptions
+type StepsDescriptors struct {
```

Comment:
Yeah, the intention since the introduction of these terms has been to use `Descriptor` as something that results from a deserialization, and `Bundle` as a collection of instances coming from instantiating objects based on the `Descriptor`.